src/components/Modals/Send/hooks/useFormSend/useFormSend.tsx

```diff
@@ -46,9 +46,26 @@ export const useFormSend = () => {
           ...accountParams
         })
 
-        const signedTx = await adapter.signTransaction({ txToSign, wallet })
+        let broadcastTXID: string | undefined
 
-        await adapter.broadcastTransaction(signedTx)
+        if (wallet.supportsOfflineSigning()) {
+          const signedTx = await adapter.signTransaction({ txToSign, wallet })
+          await adapter.broadcastTransaction(signedTx)
+        } else if (wallet.supportsBroadcast()) {
+          /**
+           * signAndBroadcastTransaction is an optional method on the HDWallet interface.
+           * Check and see if it exists; if so, call and make sure a txhash is returned
+           */
+          if (!adapter.signAndBroadcastTransaction) {
+            throw new Error('signAndBroadcastTransaction undefined for wallet')
+          }
+          broadcastTXID = await adapter.signAndBroadcastTransaction?.({ txToSign, wallet })
```

Comment:
you could define as const here as it looks like this was done to get things working as we never reassign

Reply:
We're probably going to need this return value. I intended to pass this to the toast a few lines below, but that isn't yet implemented.